src/providers/AccountService.js

```diff
@@ -25,67 +25,56 @@ export class AccountService implements AccountServiceInterface {
     this.organizationService = new OrganizationService(dispatcher);
     this.token = token;
   }
+  
+  prepRequest(request) {
+    return {
+      token: {
+        token: this.token,
+        issuer: 'Auth0'
```

Comment:
Shouldn't we have this value in some config file ? Will it be `Auth0` forever or could this change ?

Reply:
It can be changed in future according to the backend API changes